worker/symantecDistrust/symantecDistrust.go

```diff
@@ -0,0 +1,175 @@
+		SHA256: "A4310D50AF18A6447190372A86AFAF8B951FFB431D837F1E5688B45971ED1557",
+	},
+	{
+		Name:   "/C=US/O=thawte, Inc./OU=Certification Services Division/OU=(c) 2008 thawte, Inc. - For authorized use only/CN=thawte Primary Root CA - G3",
+		SHA256: "4B03F45807AD70F21BFC2CAE71C9FDE4604C064CF5FFB686BAE5DBAAD7FDD34C",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 1999 VeriSign, Inc. - For authorized use only/CN=VeriSign Class 1 Public Primary Certification Authority - G3",
+		SHA256: "CBB5AF185E942A2402F9EACBC0ED5BB876EEA3C1223623D00447E4F3BA554B65",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 1999 VeriSign, Inc. - For authorized use only/CN=VeriSign Class 2 Public Primary Certification Authority - G3",
+		SHA256: "92A9D9833FE1944DB366E8BFAE7A95B6480C2D6C6C2A1BE65D4236B608FCA1BB",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 1999 VeriSign, Inc. - For authorized use only/CN=VeriSign Class 3 Public Primary Certification Authority - G3",
+		SHA256: "EB04CF5EB1F39AFA762F2BB120F296CBA520C1B97DB1589565B81CB9A17B7244",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 2007 VeriSign, Inc. - For authorized use only/CN=VeriSign Class 3 Public Primary Certification Authority - G4",
+		SHA256: "69DDD7EA90BB57C93E135DC85EA6FCD5480B603239BDC454FC758B2A26CF7F79",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 2006 VeriSign, Inc. - For authorized use only/CN=VeriSign Class 3 Public Primary Certification Authority - G5",
+		SHA256: "9ACFAB7E43C8D880D06B262A94DEEEE4B4659989C3D0CAF19BAF6405E41AB7DF",
+	},
+	{
+		Name:   "/C=US/O=VeriSign, Inc./OU=VeriSign Trust Network/OU=(c) 2008 VeriSign, Inc. - For authorized use only/CN=VeriSign Universal Root Certification Authority",
+		SHA256: "2399561127A57125DE8CEFEA610DDF2FA078B5C8067F4E828290BFB860E84B3C",
+	},
+}
+
+type distrustedCert struct {
+	Name   string
+	SHA256 string
+}
+
+func (w runner) Run(in worker.Input, res chan worker.Result) {
+	// assume we don't trust the chain unless we find one we trust
+	trust := false
+	paths, err := in.DBHandle.GetCertPaths(&in.Certificate)
+	if err != nil {
+		w.error(res, "failed to retrieve certificate paths: %v", err)
+	}
+	trust, reason := evalPaths(paths)
+	var out string
+	if trust {
+		out = "not impacted"
+		if reason != "" {
+			out += ", but blacklisted certs found in path: " + reason
+		}
+	} else {
+		out = "impacted by blacklisted certs: " + reason
+	}
+	outB, _ := json.Marshal(out)
+	res <- worker.Result{
+		Success:    true,
+		WorkerName: workerName,
+		Errors:     nil,
+		Result:     outB,
+	}
+}
+
+func evalPaths(paths certificate.Paths) (trust bool, reason string) {
+	blacklisted, name := evalBlacklist(paths.Cert.Hashes.SHA256)
+	if blacklisted {
+		return false, name
+	}
+	for _, parent := range paths.Parents {
+		var theirReason string
+		trust, theirReason = evalPaths(parent)
```

Comment:
The nature of this is that `trust` will be set `true` or `false` depending on the final parent checked, which is not likely what we want. It seems like we want it gated: either it's `true` if `evalPaths()` ever returns true, or it's the pessimistic case of  `false` if `evalPaths()` ever returns false. I'm not sure the UI implications of these choices, but IMO it should latch.